ci/common.sh

```diff
@@ -2,26 +2,51 @@
 
 es_server="marquez.perf.lab.eng.rdu2.redhat.com"
 es_port=9200
+default_operator_image="quay.io/benchmark-operator/benchmark-operator:master"
+
+image_location=${RIPSAW_CI_IMAGE_LOCATION:-quay.io}
+image_account=${RIPSAW_CI_IMAGE_ACCOUNT:-rht_perf_ci}
+export SNAFU_IMAGE_TAG=${SNAFU_IMAGE_TAG:-snafu_ci}
+export SNAFU_WRAPPER_IMAGE_PREFIX="$image_location/$image_account"
+echo "posting container images to $image_location with account $image_account"
+export image_builder=${SNAFU_IMAGE_BUILDER:-podman}
+
+NOTOK=1
+
+# see kubernetes initialization on last line of this script
+# which will be the first thing the wrapper ci_test.sh does to it
 
 function update_operator_image() {
-  tag_name=$1
-  operator-sdk build quay.io/rht_perf_ci/benchmark-operator:$tag_name --image-builder podman
+  image_spec=$image_location/$image_account/benchmark-operator:$SNAFU_IMAGE_TAG
+  operator-sdk build $image_spec --image-builder $image_builder
 
   # In case we have issues uploading to quay we will retry a few times
   try_count=0
   while [ $try_count -le 2 ]
   do
-    if podman push quay.io/rht_perf_ci/benchmark-operator:$tag_name
+    if $image_builder push $image_spec
     then
       try_count=2
     elif [[ $try_count -eq 2 ]]
     then
-      echo "Could not upload image to quay. Exiting"
-      exit 1
+      echo "Could not upload image to $image_location. Exiting"
+      exit $NOTOK
     fi
     ((try_count++))
   done
-  sed -i "s|          image: quay.io/benchmark-operator/benchmark-operator:master*|          image: quay.io/rht_perf_ci/benchmark-operator:$tag_name # |" resources/operator.yaml
+  sed -i \
+    "s|          image: $default_operator_image|          image: $image_spec # |" \
+    resources/operator.yaml
+}
+
+function build_wrapper_image() {
+  image_spec=$1
+  wrapper_dir=$2
+  if [ "$image_builder" = "docker" ] ; then
+    docker build --tag=$image_spec -f $wrapper_dir/Dockerfile . && docker push $image_spec
+  elif [ "$image_builder" = "podman" ] ; then
+    buildah bud --tag $image_spec -f $wrapper_dir/Dockerfile . && podman push $image_spec
```

Comment:
This is causing an issue in the CI env 
```
+ buildah bud --tag quay.io/rht_perf_ci/fs-drift:snafu_ci -f fs_drift_wrapper/Dockerfile .
ci/common.sh: line 48: buildah: command not found
```
We already build the image with the operator-sdk build command on line 21. Why are we trying to build it again? It should just be a podman/docker push

Reply:
the ripsaw image is built with operator-sdk, not the snafu images.